exe/xcflushd

```diff
@@ -31,11 +31,11 @@ command :run do |c|
   c.desc 'Reporting frequency (in seconds)'
   c.flag [:f, :frequency], required: true, type: Integer, must_match: POSITIVE_N_RE
 
-  c.desc 'Number of threads for the main thread pool (min:max)'
-  c.flag [:t, :threads], default_value: 'auto', type: PositiveMinMaxInt
+  c.desc 'Number of threads for the main thread pool'
+  c.flag [:t, :threads], default_value: 'auto', type: Integer, must_match: POSITIVE_N_RE
```

Comment:
We cannot break the CLI compatibility. Just add text explaining min is not used.